src/frontend/src/utils/reimbursement-request.utils.ts

```diff
@@ -0,0 +1,21 @@
+import { ReimbursementProduct, ReimbursementRequest, wbsPipe } from 'shared';
+
+export interface ReimbursementRequestProps {
```

Comment:
this should be in the same file as the component it's typing the props of

Reply:
its typing the props of multiple components

Reply:
each component should j have their own interface, I wanna make these files as consistent and easy to follow for new ppl as possible